src/main/model/URLCommand.java

```diff
@@ -0,0 +1,76 @@
+package main.model;
+//not sure why we have this URL when there is one above it in the Curl class
+public class URLCommand {
+    private String URLs;
```

Comment:
Good question - and another good reason to have a unique name for each URL; if you find duplicates, you can remove them. ;) 

Work out logically how these will be written and their output, then see if you have overlapping info.